src/SolutionTemplate/UnoSolutionTemplate.Wizard/UnoSolutionWizard.cs

```diff
@@ -79,6 +77,38 @@ private void SetStartupProject()
 			}
 		}
 
+		private void SetUWPAnyCPUBuildableAndDeployable()
+		{
+			if (_dte?.Solution.SolutionBuild is SolutionBuild2 val)
+			{
+				try
+				{
+					var anyCpuConfig = val.SolutionConfigurations
+						.Cast<SolutionConfiguration2>()
+						.FirstOrDefault(c => c.Name == "Debug" && c.PlatformName == "Any CPU");
+
+					foreach (SolutionConfiguration2 solutionConfiguration2 in val.SolutionConfigurations)
+					{
+						foreach (SolutionContext solutionContext in anyCpuConfig.SolutionContexts)
+						{
+							if (solutionContext.ProjectName.EndsWith(".UWP.csproj"))
+							{
+								solutionContext.ShouldBuild = true;
+								solutionContext.ShouldDeploy = true;
+							}
+						}
+					}
+				}
+				catch (Exception)
+				{
+				}
+			}
+			else
+			{
+				throw new InvalidOperationException();
```

Comment:
There's probably no reason to make this fail. If it happens, let's let it continue.

Reply:
@jeromelaban the other methods `SetDefaultConfiguration` and `SetStartupProject` also throw in this case, should I remove the throws everywhere, so the solution creation doesn't fail?

Reply:
Right, let's keep it that way for now then. It's unlikely to fail in this case, and if it would, lots more would fail anyways.